zulipterminal/model.py

```diff
@@ -200,22 +205,19 @@ def react_to_message(self,
         # FIXME Only support thumbs_up for now
         assert reaction_to_toggle == 'thumbs_up'
 
-        endpoint = 'messages/{}/reactions'.format(message['id'])
         reaction_to_toggle_spec = dict(
             emoji_name='thumbs_up',
-            reaction_type='unicode_emoji',
-            emoji_code='1f44d')
+            emoji_code='1f44d',
+            reaction_type='unicode_emoji',  # FIXME Should be emoji_type?
```

Comment:
I think `reaction_type` is used for differentiating between 'unicode_emoji` and user create emojis.

Reply:
I used `reaction_type` originally, just for the thumbs-up, and it works - but the python API method seems to indicate `emoji_type=unicode_emoji`. I just checked the backend and it has our approach, so I'll remove the comment and leave it as-is. We can always add an update if the API changes.